docs/website-feedback.md

```diff
@@ -0,0 +1,99 @@
+# Website feedback
+
+Website pages can be configured to display forms that solicit feedback from
+visitors, for example asking whether or not page information was useful and
+accurate.
+
+User-submitted feedback is stored in the Django database. There are several
+ways to access and export this data.
+
+## Exporting feedback using Wagtail
+
+Users with appropriate permissions have access to an "Export Feedback" tool
+in Wagtail. This tool generates a ZIP file containing submitted feedback
+between two user-specified dates.
+
+![Screenshot of the Wagtail tool for exporting feedback](img/export-feedback.png)
+
+Feedback is automatically grouped into three segments, each stored as a CSV:
+
+- Feedback submitted on Ask CFPB pages (both English and Spanish)
+- Feedback submitted on Buying a House pages
+- Feedback submitted on all other website pages
+
+In order to access this part of the Wagtail admin, users must be granted the
+"Can export feedback from the Wagtail admin" permission. This can be enabled
+or disabled for different Wagtail groups in the Groups area of the admin.
+
+## Exporting feedback from the command line
+
+For more control over what feedback to export, the
+[`export_feedback`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/management/commands/export_feedback.py)
+Django management command can be used. This command generates a CSV file
+containing feedback for a specified set of pages and/or dates.
+
+To export all website feedback for all time, use this command:
+
+```sh
+cfgov/manage.py export_feedback --filename output.csv
+```
+
+(Alternatively, exclude the `--filename` parameter to send the output to the
+console.)
+
+To export feedback for a specific Wagtail page, provide its slug. Specified
+pages must have slugs that are unique across the website.
+
+```sh
+cfgov/manage.py export_feedback ask_cfpb --filename ask_cfpb.csv
+```
+
+Specify multiple slugs to export feedback for multiple pages:
+
+```sh
+cfgov/manage.py export_feedback ask_cfpb owning-a-home --filename ask_and_bah.csv
+```
+
+To invert the selection, and export feedback _except_ for those pages
+specified, use the `--exclude` parameter:
+
+```sh
+cfgov/manage.py export_feedback ask_cfpb --exclude --filename everything_except_ask.csv
```

Comment:
This now works, but I would like the command to have `--exclude` first, which seems more clear to me.

```suggestion
cfgov/manage.py export_feedback --exclude ask-cfpb --filename everything_except_ask.csv
```

Also, `ask_cfpb` is not a valid slug; it should be `ask-cfpb`. Updated in my suggestion on this line, but please update in all other examples.

Reply:
Addressed in 60a0d23.